airgun/views/subscription.py

```diff
@@ -158,6 +158,12 @@ class ManageManifestView(BaseLoggedInView):
 
     @View.nested
     class manifest(SatTab):
+        alert_message = Text(
+            '//*[@id="manifest-history-tabs-pane-1"]/div/h3//following-sibling::div[@aria-label="Warning Alert" or @aria-label="Danger Alert"]'
+        )
+        expire_header = Text('//*[@id="manifest-history-tabs-pane-1"]//following::div/h4')
+        expire_message = Text('//*[@id="manifest-history-tabs-pane-1"]//following::div/div[2]/span')
+        expire_date = Text('//*[@id="manifest-history-tabs-pane-1"]/div[1]/div[3]/div[2]')
```

Comment:
Please specify the correct tag name instead of using '*' in the list of tag names.